docs/user/ai_model_serving.md

```diff
@@ -0,0 +1,482 @@
+# AI Model Serving on MicroShift
+
+AI Model Serving on MicroShift is a platform for serving AI models.
+It includes limited subset of Red Hat OpenShift AI (RHOAI): [kserve](https://github.com/opendatahub-io/kserve) and ServingRuntimes.
+Now you can train your models in the cloud and serve them on the edge.
+
+MicroShift is intended for the edge (small footprint, low resources),
+therefore kserve is configured to use a "Raw Deployment" mode which means that:
+- Kubernetes Deployments and Services will be created, and
+- neither Service Mesh (Istio) nor Serverless (Knative) are required.
```

Comment:
If this is a user doc, I'm not sure how to interpret this info, mostly the first bullet.

Reply:
Yeah. I wrote this doc from a perspective of someone already knowing kserve so wanted to write out the differences, but maybe we need to show in simpler way how we intend it to be used on microshift